src/components/table/table-row/table-row.style.js

```diff
@@ -0,0 +1,200 @@
+import { css } from 'styled-components';
+import StyledTableCell from '../table-cell/table-cell.style';
+import StyledTableHeader from '../table-header/table-header.style';
+import { isClassic } from '../../../utils/helpers/style-helper';
+import { THEMES } from '../../../style/themes';
+
+const StyledTableRow = css`
+  .carbon-table-row {
+    ${applyRowStyling}
+  }
+`;
+
+function applyRowStyling(props) {
+  const { theme } = props;
+  return css`
+    .custom-drag-layer {
+      & {
+        background-color: #002E41;
+        cursor: grabbing;
+        cursor: -moz-grabbing;
+        cursor: -webkit-grabbing;
+        display: block;
+      }
+
+      ${StyledTableCell} {
+        background-color: #002E41;
+      }
+
+      .configurable-item-row__content-wrapper {
+        visibility: visible;
+      }
+    }
+
+    &:first-child ${StyledTableHeader} {
+      &:first-child {
+        border-radius: 0px 0 0 0;
+      }
+
+      &:last-child {
+        border-radius: 0 0px 0 0;
+      }
+    }
+
+    ${isClassic(theme) || theme.name === THEMES.base ? applyClassicStyling : applyModernStyling}
```

Comment:
isn't the base theme a part of the modern styling?

Reply:
The table has a lot of cross over between the new and old themes, so this just defaults it to classic if there no theme is passed